File: src/Estudo.UI/src/app/models/lead.model.ts
export class Lead {
    id: string;
    firstName: string;
    lastName: string;
    dateCreated: string;
    phoneNumber: string;
    email: string;
    suburb: string;
    category: string;
    jobId: string;
    description: string;
    price: number;

    constructor(id: string, firstName: string, lastName: string, dateCreated: string, phoneNumber: string, email: string,
        suburb: string, category: string, jobId: string, description: string, price: number) {
            this.id = id,
            this.firstName = firstName,
            this.lastName = lastName,
            this.dateCreated = dateCreated,
            this.phoneNumber = phoneNumber,
            this.email = email,
            this.suburb = suburb,
            this.category = category,
            this.jobId = jobId,
            this.description = description,
            this.price = price
    }
}

File: src/Estudo.Api/Domain/Entities/v1/LeadModel.cs
using Domain.Enum.v1;

namespace Domain.Entities.v1
{
    public class LeadModel
    {
        public Guid Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime DateCreated { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public string Suburb { get; set; }
        public long JobId { get; set; }
        public string Category { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public StatusEnum Status { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string ModifiedUser { get; set; }
    }
}